### DISCUSSION TITLE
Groups and Fields visual - icons/images/colors

### DISCUSSION THREAD

**Discussion by justdudeone** [pods-framework/pods#6805] [Ideas]
Hello,
For a huge Pods data with >30 fields (for example 100 fields) - will be very nice to have possibility to add/set a custom icons/images/colors/etc to each Group and each Field which will be displayed at frontend - such visual distinguish can significantly rise overall readability!

**Comment by JoryHogeveen** [comment]
Hi @justdudeone 
Could you provide us with examples of what you mean?

**Comment by justdudeone** [comment]
https://motors.stylemixthemes.com/classified/listings/bmw-m5/ - main tech specs (Taxonomy) - each with individual icon;
https://motors.stylemixthemes.com/listings/tesla-roadster/ - tab "Technical" - each group have individual icon.

To make it you need to add to all Pods Groups and Pods Fields new simple possibility to set an color/icon/image and then display them at frontend near the Field/Group name, or even with nice option "display Field/Group as: name, color, color(circle)+name, icon, icon+name, image, image+name".

**Comment by JoryHogeveen** [comment]
This can be done using templates. You can set your own templates and insert the icons where you want.

**Comment by justdudeone** [comment]
Suppose it require manual CSS coding etc? It just break overall nice Pods GUI conception!

**Comment by JoryHogeveen** [comment]
Yep, it will require CSS etc.
However, that is the case with all front-end customizations.
Pods does not do any front-end CSS.

**Comment by justdudeone** [comment]
Visual distinguish of Pods Group/Field by Image/Icon/Color assign - is not only for frontend, it's also VERY useful at huge databases BACKEND for easy input, avoid mess, mix up, etc

**Comment by JoryHogeveen** [comment]
For the backend you can actually already add dashicons as label prefixes. You can manually enter the icon HTML inside the field label to see it's icon in the backend.
For front-end however this is separate. You can then even use a totally different icon library.

**Comment by justdudeone** [comment]
Not for frontend only, not for backend only, not by manual crutches/kludges/tricks, etc.

It just can be done right - for backend and for frontend and native - by adding icons/images/colors to Groups and Fields.

**Comment by JoryHogeveen** [comment]
I've added the correct addon/feature labels to this issue. If there is enough interest in such a feature we will make resources available to build this.
However, to be honest, I don't expect this to be anytime soon. Are are most welcome to contribute to Pods if you need this feature earlier!

**Comment by justdudeone** [comment]
I'm not a web dev (I'm experienced dev for Win) and don't know WP enough to develop anything here fast&effective and can't spend too much time to learn it from scratch, unfortunately.
I'm extremely generous man - I help to tons of free projects to become better and to many people who really need help. Unfortunately I live in a backward country (also with protracted economic crisis and mass imitation of work and arranging good jobs only by acquaintance for budget stealing etc) - thus as honest person without acquaintance I get too low income now to pay at your (USA/Europe) taxes something like $50-100/hr = $8'000-15'000/months - its totally insane money for me now. I can pay overall just $100, maybe $200, not more, sorry. :'(
